### PR TITLE
Fix functional unit test after hosting changes

### PR DESCRIPTION
**Pull request by pakrym**
@Tratcher

### CODE REVIEW

**Comment by Tratcher** [comment]
@HaoK

**Comment by Tratcher** [comment]
This probably needs to be a production dependency, not a test dependency.

**Comment by pakrym** [comment]
:up: :date:

**Comment by HaoK** [comment]
:shipit:

File: src/Microsoft.AspNet.Identity/IdentityServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Internal;
```
Comment (HaoK) [inline src/Microsoft.AspNet.Identity/IdentityServiceCollectionExtensions.cs]: Why do we need both Http and Internal?

File: src/Microsoft.AspNet.Identity/IdentityServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -49,6 +51,8 @@ public static class IdentityServiceCollectionExtensions
                 options.SignInScheme = new IdentityCookieOptions().ExternalCookieAuthenticationScheme;
             });
 
+            // Hosting doesn't add IHttpContextAccessor byt default
```
Comment (HaoK) [inline src/Microsoft.AspNet.Identity/IdentityServiceCollectionExtensions.cs]: nit: typo

File: src/Microsoft.AspNet.Identity/IdentityServiceCollectionExtensions.cs (hunk 1 above)
Comment (pakrym) [inline src/Microsoft.AspNet.Identity/IdentityServiceCollectionExtensions.cs]: Yep  HttpContextAccessor  is in internal

**Pull request merged**